### PR TITLE
[feat] Create login page

### PR DESCRIPTION
**Pull request by Hamdrive**
Created login page, similar layout to sign up page.

### CODE REVIEW

**Review by AnuragSahu11 — APPROVED** [review]
Looks good, you have used very unique colors

File: css/login.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  min-height: 80vh;
+  position: relative;
+}
+
+.login-modal {
+  max-width: 30rem;
+  margin: auto;
+  position: relative;
+}
+
```
Comment (lannisterrr) [inline css/login.css:22]: code seems fine.

**Review by lannisterrr — APPROVED** [review]

File: css/login.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+.w-100 {
+  width: 100%;
+}
+
+.form-border {
+  border: 2px solid var(--primary-dark-color);
+  border-radius: var(--radius-extra-large);
+}
+
+.mb-1 {
```
Comment (Varun-28) [inline css/login.css:32]: everything seems clean and good 👍

**Comment by Hamdrive** [comment]
Merging to dev

**Pull request merged**